src/main/java/domain/LottoUnitWinnings.java

```diff
@@ -0,0 +1,35 @@
+package domain;
+
+import java.util.Arrays;
+
+public enum LottoUnitWinnings {
+    THREE(3, new Price(5000L)),
+    FOUR(4, new Price(50000L)),
+    FIVE(5, new Price(1500000L)),
+    SIX(6, new Price(2000000000L));
+
+    private final Integer equalCount;
+    private final Price winningPrice;
+
+    LottoUnitWinnings(int equalCount, Price winningPrice) {
+        this.equalCount = equalCount;
+        this.winningPrice = winningPrice;
+    }
+
+    public Price getWinningPrice() {
+        return this.winningPrice;
+    }
+
+    public Integer getEqualCount() {
+        return this.equalCount;
+    }
+
+
+    public static Price findWinningPrice(long equalCount) {
+        return Arrays.stream(LottoUnitWinnings.values())
+                .filter(lottoUnitWinning -> lottoUnitWinning.getEqualCount() == equalCount)
+                .map(lottoUnitWinning -> lottoUnitWinning.getWinningPrice())
+                .findAny()
+                .orElseGet(() -> new Price(0L));
```

Comment:
정적 팩토리 메소드를 사용하신다면 이런 부분들은 가독성 개선 효과를 볼 수 있습니다 😄 
```suggestion
                .orElseGet(() -> Price.zero());
```

도움 되실만한 내용 공유 드립니다 🙇 
> https://johngrib.github.io/wiki/pattern/static-factory-method/

Reply:
감사합니다! 좀 더 명확하게 표현이 가능하네요